Fix final styling

alvarezGarciaMarcos opened the pull request:
This PR holds all the final modifications done to the styling for this last deliverable.

Commit:
Modifying styling of notifications

Commit:
Merged

Commit:
Added styling for notifications

Commit:
Added styling for Comments and slideshow

Commit:
Add styling for the Settings page

Commit:
More styling and fixing

I added our favicon to the page, and also fixed some errors in the
Settings tests regarding some changes done to the settings page

lamasumas reviewed (approved):
Look awesome the new style of  the pages

themrcesi committed:
Route is cleared when view is changed.

themrcesi committed:
Also in dropdowns

themrcesi committed:
Delete solid

Now when a route is deleted all its linked data: comments file and resources are also deleted.

themrcesi reviewed (changes requested):
Just take a look at the button comment, and once it´s fixed ready to merge I think

Commit:
Added styling to routeDetails so everything is shown

Commit:
Merge branch 'fix-final-styling' of https://github.com/Arquisoft/viade_en1b into fix-final-styling
